packages/fx-core/src/core/index.ts

```diff
@@ -936,6 +936,24 @@ export class FxCore implements Core {
     return await ctx!.solution!.listAllCollaborators!(ctx!.solutionContext!);
   }
 
+  @hooks([
+    ErrorHandlerMW,
+    ConcurrentLockerMW,
+    ProjectSettingsLoaderMW,
+    EnvInfoLoaderMW(true),
+    ContextInjectorMW,
+  ])
+  async getSelectedEnv(
```

Comment:
What's the return value for non multi env?

Reply:
Thanks! Update to throw error for non multi-env.